packages/@ember/test/index.ts

```diff
@@ -17,7 +17,7 @@ registerWaiter = testingNotAvailableMessage;
 unregisterHelper = testingNotAvailableMessage;
 unregisterWaiter = testingNotAvailableMessage;
 
-export function registerTestImplementaiton(impl: typeof EmberTesting) {
+export function registerTestImplementation(impl: typeof EmberTesting) {
```

Comment:
This is newly-added private API appearing for the first time in this beta. Seems a good time to fix my typo.